frontend/public/i18n.js

```diff
@@ -59,6 +59,14 @@ i18n
       'rhoas-plugin',
       'topology',
       'tour',
+      'user',
```

Comment:
I don't think we should be introducing any of these new namespaces, but putting everything into `public`. @rebeccaalpert  can you confirm?

Reply:
@kyoto  and @rebeccaalpert ,
I changed the codes to put everything into public. 
Thanks